schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -480,6 +480,9 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-09-01/Microsoft.Network.json#/resourceDefinitions/dnsZones_SOA" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-09-01/Microsoft.Network.json#/resourceDefinitions/dnsZones_SRV" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-09-01/Microsoft.Network.json#/resourceDefinitions/dnsZones_TXT" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2016-03-01/Microsoft.Insights.json#/resourceDefinitions/alertrules" },
```

Comment:
>{ "$ref": "https://schema.management.azure.com/schemas/2016-03-01/Microsoft.Insights.json#/resourceDefinitions/alertrules" }, [](start = 18, length = 125)

This is not there in master branch. I guess you didn't merge correctly. Please resolve this.